### ISSUE TITLE
Is there more documentation available?

### ISSUE THREAD

**Issue by izznogooood** [fiduswriter/django-npm-mjs#3]
This looks just like the thing I've been looking for but I can't seem to get this working.

Django 2.2, Python 3.7
```
(venv) anders@yoga:~/dev/npm_mjs$ ./manage.py transpile
Unknown command: 'transpile'
```
I also get a TemplateSyntaxError:
'transpile' is not a registered tag library.

I have followed the quick start to the letter.

It is also not clear to me how you grab the modules? Is it by the import statements made in a linked mjs file in a template? 

Do I need a node_modules folder in every app?

I've pushed the test here:
https://github.com/izznogooood/npm_mjs

**Comment by johanneswilm** [comment]
> Do I need a node_modules folder in every app?

No, just a `packages.json`. It should then put together one common node_modules automatically.
